contracts/MetadataViews.cdc

```diff
@@ -14,6 +14,32 @@ import "ViewResolver"
 ///
 access(all) contract MetadataViews {
 
+    /// Function to resolve a contract view based on a type identifier String
+    /// and view type. Borrows the contract as &{ViewResolver} and 
+    /// then calls resolveContractView for the specified type and view.
+    ///
+    /// @param resourceType: An optional resource type to return views for
+    /// @param viewType: The Type of the desired view.
+    /// @return A structure representing the requested view. If anything failed, returns nil
+    ///
+    access(all) fun resolveContractViewFromTypeIdentifier(
+        resourceTypeIdentifier: String,
```

Comment:
I don't feel super strongly about this but it seems like for the sake of consistency at least the param should be `resourceType: Type` given `viewType: Type` and `resolveContractView()` param types. If so, the method needs renaming.

If `*Identifier` makes more sense based on how this will be called, then we should update `@param` to match the parameter name and type.

Reply:
oh yeah, I just forgot to update the comment. I think it is better to do it as the identifier since it saves more code